fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -69,6 +69,18 @@
     FedoraResource getContainer();
 
     /**
+     * Get the TimeMap/LDPCv of this resource
+     * @return the container for TimeMap/LDPCv of this resource
+     */
+    FedoraResource getLDPCv();
```

Comment:
Actually, instead of adding the `TimeMap` functionality to the existing `FedoraResource`, it may be cleaner to create the new class `FedoraTimeMap.java`.

Additionally, I believe we were planning on the timemap resource to be named `fcr:versions` and to have requests to timemaps be intercepted by a Jersey path selector:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersioning.java#L73